tests/lib/snaps/test-snapd-profiler/profiler.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
It'd be nice if we could find a way to share the file in the source tree.

Reply:
What did you mean by share?

Reply:
Having only one copy of the file in snapd source tree. I recall we have the same problem with shell wrappers for snap, there's a bunch of files under `tests/lib/snaps/*/bin` that are copies.

Reply:
Totally agree on this. I think I could create a new PR with that change fixing all the repeated files. Are you ok with this?

Reply:
Yes, a follow up sounds ok.